readme.md

```diff
@@ -23,31 +50,34 @@ import { Widget } from "@uploadcare/react-widget";
 <Widget publicKey="demopublickey" />;
 ```
 
-## Available props
+## Configuration
 
-All the props except for the listed below are passed down to the widget. See [widget documentation](https://uploadcare.com/docs/file_uploads/widget/options/#options) for the details.
+### Component configuration
 
-### `value`: `string`
+#### `value`: `string`
 
-Set a file/group UUID or a [CDN link](https://uploadcare.com/docs/delivery/) as a value.
+Set a file/group UUID or a [CDN URL][delivery-docs] as a value.
```

Comment:
So I already know what file/group UUIDs are, but am not yet aware of the CDN URLs?
pls add the reference to `[file/group UUID](<source>)`